tests/integration/returners/test_local_cache.py

```diff
@@ -110,6 +110,28 @@ def test_empty_jid_dir(self):
         EMPTY_JID_DIR.append(new_jid_dir)
         os.makedirs(new_jid_dir)
 
+        # This needed due to a race condition in Windows
+        # `os.makedirs` hasn't released the handle before
+        # `local_cache.clean_old_jobs` tries to delete the new_jid_dir
+        if salt.utils.is_windows():
+            import time
+            lock_dir = new_jid_dir + '.lckchk'
+            tries = 0
+            while True:
+                tries += 1
+                if tries > 10:
+                    break
+                # Rename the directory and name it back
+                # If it fails, the directory handle is not released, try again
+                # If it succeeds, break and continue test
+                try:
+                    os.rename(new_jid_dir, lock_dir)
+                    time.sleep(1)
+                    os.rename(lock_dir, new_jid_dir)
+                    break
+                except WindowsError:
```

Comment:
Probably because the lint tests run on a linux box that doesn't have WindowError...